CMakeLists.txt

```diff
@@ -679,6 +692,12 @@ if (HAVE_KERNARG_PRELOAD)
   target_link_options(rccl PRIVATE -Xoffload-linker -mllvm=-amdgpu-kernarg-preload-count=16)
 endif()
 
+if(ENABLE_MSCCLPP)
+  message(STATUS "Building MSCCLPP")
```

Comment:
We may not actually be building MSCCL++, so move this message into MSCCLPP.cmake and correct it.

Reply:
```suggestion
```